docs/common/registration.md

```diff
@@ -39,7 +39,8 @@ To register your contact information with the OSG Consortium, follow the instruc
 Registering Resources
 ---------------------
 
-An OSG resource is a host that provides grid services, e.g. Compute Entrypoints, storage endpoints, or perfSonar hosts.
+An OSG resource is a host that provides services to OSG campuses and collaborations;
+some examples are Compute Entrypoints, storage endpoints, or perfSonar hosts.
```

Comment:
It should be “perfSONAR” throughout — that’s how they write it.

Reply:
Fixed